util.py

```diff
@@ -43,6 +44,24 @@
 NumericTypes = (float, int, long)
 
 
+def log_subprocess(func):
```

Comment:
Could you put this function and `import subprocess` in `utils/` ?
I think we are slowly deprecating `util.py` in favor of `utils/`, more precise/beauteous.